packages/web/src/pages/admin/expoJudgingSession/[id]/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { NextPage } from 'next';
+import { Box, UnorderedList, ListItem, Heading, Flex, Text } from '@chakra-ui/react';
+import { PageContainer } from '../../../../components/layout/PageContainer';
+import { useExpoJudgingSessionFetcher } from '../../../../pageUtils/expoJudgingSession/[id]/useExpoJudgingSessionFetcher';
+import {
+  ExpoJudgingSessionResult,
+  fetchExpoJudgingSessionResults,
+} from '../../../../pageUtils/expoJudgingSession/[id]/fetchExpoJudgingSessionResults';
+import { openErrorToast } from '../../../../components/utils/CustomToast';
+
+const ExpoJudgingSessionDetails: NextPage = () => {
+  const { expoJudgingSession } = useExpoJudgingSessionFetcher();
+  const [expoJudgingSessionResults, setExpoJudgingSessionResults] =
+    React.useState<ExpoJudgingSessionResult[]>();
+  React.useEffect(() => {
```

Comment:
```suggestion

  React.useEffect(() => {
```

Reply:
that's not me, that is prettier.

Reply:
I merged the commit from the UI and Prettier is fine with it